astropy/cosmology/io/mrt.py

```diff
@@ -0,0 +1,116 @@
+from astropy.cosmology.connect import readwrite_registry
+from astropy.cosmology.core import Cosmology
+from astropy.table import QTable
+from astropy.table.serialize import represent_mixins_as_columns
+
+from .table import from_table, to_table
+
+
+def read_mrt(filename, index=None, *, move_to_meta=False, cosmology=None, **kwargs):
+    """Read a |Cosmology| from an MRT file.
+
+    Parameters
+    ----------
+    filename : path-like or file-like
+        The file from which to read the MRT data.
+    index : int or str or None, optional
+        Needed to select the row in tables with multiple rows. ``index`` can be
+        an integer for the row number or, if the table is indexed by a column,
+        the value of that column. If the table is not indexed and ``index`` is a
+        string, the "name" column is used as the indexing column.
```

Comment:
I don't think MRT tables preserve which column is an indexing column.

```suggestion
    index : int or str or None, optional
        Needed to select the row in tables with multiple rows. ``index`` can be
        an integer for the row number or, if ``index`` is a
        string, the "name" column is used as the indexing column.
```